vignettes/wpgsd_corr_example.Rmd

```diff
@@ -18,7 +19,7 @@ $$Corr(Z_{ik},Z_{i'k'})=\frac{n_{i \wedge i',k \wedge k'}}{\sqrt{n_{ik}*n_{i'k'}
 
 ## Examples
 
-In a 2-arm controlled clinical trial example with one primary endpoint, there are 3 patient populations defined by the status of two biomarkers A and B:
+In a 2-arm controlled clinical trial example with one primary endpoint (@anderson_unified_2022), there are 3 patient populations defined by the status of two biomarkers A and B :
```

Comment:
Please be more specific where the example is from, say, which section, which example. Some explanations like, "this  example is borrowed from xxx paper" is helpful.